### ISSUE TITLE
Seems like ES7 decorator syntax breaks the trucker

### ISSUE THREAD

**Issue by SeanChenU** [davidmfoley/node-trucker#15]
running this command:
`trucker --dry-run js/components/Login.js js/components/LoginPage.js`

results in:
```
js/components/Login.js  ->  js/components/LoginPage.js
...skipped...
3: - import { Component } from 'react'
3: + import { Component } from 'react'./react-native
4: - import {StyleSheet, AppRegistry, Text, View, TouchableHighlight, TextInput } from 'react-native';
4: + import {StyleSheet, AppRegistry, T./react-navigationableHighlight, TextInput } from 'react-native';
6: - import loginStore from '../stores/loginStore'
6: + import loginStore from ./mobx-react/nativetore'
```

I use to decorator syntax like this:
```
@observer
export default class Login extends Component {
  constructor(props) {
    super(props);
  }

...skipped...

```

After removing the decorator syntax, it works fine

**Comment by davidmfoley** [comment]
I know it's like 8 months later but I think I just found the cause of this in investigating another bug: #17

**Comment by davidmfoley** [comment]
I think this is fixed -- kind of -- in 0.7.5.

When trucker can't parse a file using babylon, it falls back to a regex-based approach that works for many files but doesn't cover all possible syntax. A *real* fix would be to update the babel parsing stack that trucker uses so that it doesn't fall back on the regex nonsense at all... I'll do that in the next version. In the meantime, try 0.7.5 and see if that works for you.
